SomeLogicLibrary/Class/TestSErvice.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Castle.Components.DictionaryAdapter;
+using Castle.MicroKernel.Registration;
+using Castle.Windsor;
+using Castle.Windsor.Installer;
+using SomeDataLibrary.Class;
+using SomeDataLibrary.Interface;
+using SomeDataLibrary.Model;
+using SomeLogicLibrary.Interface;
+
+namespace SomeLogicLibrary.Class
+{
+    public class TestService :ITestService
+    {
+        private  IWindsorContainer container;
+        private readonly IUserRepository userRepository;
+        private readonly ICompanyRepository companyRepository;
+        private INotificator[] notificators;
+
+        public TestService(IWindsorContainer container)
+        {
```

Comment:
You should inject dependencies in constructor instead od container. Please read Mark Seemann book https://www.manning.com/books/dependency-injection-in-dot-net